<launch>

    <include file="$(find-pkg-share turtle_brick)/show_turtle.launch.xml">
        <arg name="use_jsp" value="none"/>
    </include>

    <node pkg="turtlesim" exec="turtlesim_node">
        <param name="holonomic" value="true"/>
    </node>

    <node pkg="turtle_brick" exec="turtle_robot">
        <remap from="/cmd_vel" to="turtle1/cmd_vel"/>
    </node>

    <node pkg="turtle_brick" exec="arena">
    </node>

</launch>
